board/safety.h

```diff
@@ -82,9 +82,20 @@ int safety_fwd_hook(int bus_num, CANPacket_t *to_fwd) {
 }
 
 bool get_longitudinal_allowed(void) {
+  // No longitudinal control when overriding with gas. Brake is allowed when pre-enabling at a standstill
   return controls_allowed && !gas_pressed_prev;
 }
 
+bool get_lateral_allowed(void) {
+  // No steering allowed while pre-enabling at a standstill with brake
+  return controls_allowed && !brake_pressed_prev;
+}
```

Comment:
TODO: get rid of get_lateral_allowed and make common check functions for tesla, nissan, and ford